crates/ruff_linter/resources/test/fixtures/airflow/AIR302_names.py

```diff
@@ -282,6 +292,9 @@
 # airflow.utils.dag_parsing_context
 get_parsing_context
 
+# airflow.utils.db
+create_session
+
```

Comment:
This is not being considered (refer to my other comment) as seen in the snapshot.

Can you make sure to verify the snapshot changes? I know they're a bit hard to read when the changes are in the middle of the file. It might be easier to add new test cases at the end of the file to make snapshot changes easier to read and then moving it back to the relevant place and update the "Test plan" in the PR description to mention if you've done this.

Reply:
> This is not being considered (refer to my other comment) as seen in the snapshot.
Hi, I apologize for missing this one.

> It might be easier to add new test cases at the end of the file to make snapshot changes easier to read and then moving it back
This is an Great idea. I used to perform text searches on the git diff, but it's easy to overlook. I'll take care of it afterward. Thank you so much!